### PR TITLE
8 implementation of insertion sort algorithm

### PR DESCRIPTION
**Pull request by VladaKmets**
+ some other tasks

### CODE REVIEW

File: src/main/java/com/vlada/tasks/makeEqual/MakeEqual.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        Scanner keyboard = new Scanner(System.in);
+        System.out.println("Please, enter the number of elements of a massive");
+        String numberOfElements = keyboard.nextLine();
+        int numberOfIntegers = Integer.valueOf(numberOfElements);
+        Integer[] arrayOfIntegers = {};
+
+        for (int i = 0; i < numberOfIntegers; i++) {
+            System.out.println("Please, enter an integer");
+            String arrayOfInts = keyboard.nextLine();
+            arrayOfIntegers[i] = Integer.valueOf(arrayOfInts);
```
Comment (MaksymKmets) [inline src/main/java/com/vlada/tasks/makeEqual/MakeEqual.java:18]: How it works? You have initiated an empty array above...

File: src/main/java/com/vlada/tasks/monkeycount/MonkeyCount.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package com.vlada.tasks.monkeycount;
+
+public class MonkeyCount {
+    public static void countMonkeys() {
+     /*   Scanner scanner = new Scanner(System.in);
```
Comment (MaksymKmets) [inline src/main/java/com/vlada/tasks/monkeycount/MonkeyCount.java:5]: Never leave code in comments. It is useless and even confusing

File: src/main/java/com/vlada/tasks/makeEqual/MakeEqual.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        }
+        System.out.println(Arrays.toString(arrayOfIntegers));
+        System.out.println("Please, enter a number to make equal every integer in array to");
+        Integer integerToMakeEqualTo = keyboard.nextInt();
+        System.out.println("Please, enter a number to add/extract");
+        Integer toAddOrExtract = keyboard.nextInt();
```
Comment (MaksymKmets) [inline src/main/java/com/vlada/tasks/makeEqual/MakeEqual.java:24]: Always close Scanner objects in finally blocks! Check everywhere

File: src/main/java/com/vlada/tasks/makeEqual/MakeEqual.java (hunk 1 above)
Comment (VladaKmets) [reply src/main/java/com/vlada/tasks/makeEqual/MakeEqual.java:18]: Fixed

File: src/main/java/com/vlada/tasks/makeEqual/MakeEqual.java (hunk 3 above)
Comment (VladaKmets) [reply src/main/java/com/vlada/tasks/makeEqual/MakeEqual.java:24]: Done

File: src/main/java/com/vlada/tasks/monkeycount/MonkeyCount.java (hunk 2 above)
Comment (VladaKmets) [reply src/main/java/com/vlada/tasks/monkeycount/MonkeyCount.java:5]: Done

**Pull request merged**